spec/01-unit/05-utils_spec.lua

```diff
@@ -437,6 +437,24 @@ describe("Utils", function()
   end)
 
   describe("hostnames and ip addresses", function()
+    it("splits #port number", function()
+      for _, case in ipairs({
+        {'', {'', nil} },
+        {'localhost', {'localhost', nil}},
+        {'localhost:', {'localhost', nil}},
+        {'localhost:80', {'localhost', 80}},
+        {'localhost:23h', {'localhost:23h', nil}},
+        {'localhost/24', {'localhost/24', nil}},
+        {'::1', {'::1', nil}},
+        {'[::1]', {'::1', nil}},
+        {'[::1]:', {'::1', nil}},
+        {'[::1]:80', {'::1', 80}},
+        {'[::1]:80b', {'[::1]:80b', nil}},
+        {'[::1]/96', {'[::1]/96', nil}},
+      }) do
+        assert.same(case[2], {utils.split_port(case[1])})
```

Comment:
style: missing spaces in the table notation (`{ ... }`)

Reply:
done